app/components/select-tokens.tsx

```diff
@@ -0,0 +1,218 @@
+import React from "react";
+import { Open_Sans } from "next/font/google";
+import btc from "../../public/coin-logos/btc-logo.png";
+import eth from "../../public/coin-logos/eth-logo.png";
+import strk from "../../public/coin-logos/strk-logo.png";
+import alt from "../../assets/alt.png";
+import type { StaticImageData } from "next/image";
+
+import Image from "next/image";
+const openSans = Open_Sans({ subsets: ["latin"] });
+
+type Coin = {
+  key: string;
+  imgLink: StaticImageData | string;
+  coinName: string;
+  coinSymbol: string;
+};
+
+const Selecttokens = () => {
+  const coins: Coin[] = [
+    { key: "bitcoin", imgLink: strk, coinName: "Bitcoin", coinSymbol: "BTC" },
+    { key: "ethereum", imgLink: eth, coinName: "Ethereum", coinSymbol: "ETH" },
+    { key: "dogecoin", imgLink: eth, coinName: "Dogecoin", coinSymbol: "DOGE" },
+    { key: "litecoin", imgLink: btc, coinName: "Litecoin", coinSymbol: "LTC" },
+    { key: "ripple", imgLink: alt, coinName: "Ripple", coinSymbol: "XRP" },
+    { key: "cardano", imgLink: alt, coinName: "Cardano", coinSymbol: "ADA" },
+    { key: "polkadot", imgLink: alt, coinName: "Polkadot", coinSymbol: "DOT" },
+    { key: "solana", imgLink: alt, coinName: "Solana", coinSymbol: "SOL" },
+    {
+      key: "binanceCoin",
+      imgLink: alt,
+      coinName: "Binance Coin",
+      coinSymbol: "BNB",
+    },
+    {
+      key: "avalanche",
+      imgLink: alt,
+      coinName: "Avalanche",
+      coinSymbol: "AVAX",
+    },
+  ];
+
+  return (
+    <div className=" w-full  pt-[180px] items-center justify-center flex">
+      <div
+        className={`shadow-lg rounded w-[350px] sm:w-[800px] ${openSans.className}  h-[660px] flex justify-center flex-col items-center`}
+      >
+        <div className=" w-[80%] flex items-center justify-between">
+          <p className="none sm:block"></p>
+          <p className="font-[600]  text-center sm:text-left text-[20px] sm:text-[24px] ">
+            Select token(s) to auto-swap from
+          </p>
+          <div className="none sm:block">
+            <CloseButton />
+          </div>
+        </div>
+        <br />
+        <p className="text-center font-normal text-[14px] sm:text-[16px]  leading-[22px] w-[318px] sm:w-[620px] ">
+          You can select multiple tokens to auto-swap from here. You can select
+          how many percent of that token should be auto-swapped whenever you get
+          funded.
+        </p>
+        <br />
+
+        <form className="mx-auto flex items-center justify-center flex-col">
+          <div className="relative bg-[#100827] px-4 py-2 flex items-center justify-between w-[318px] sm:w-[704px] rounded-full">
+            <input
```

Comment:
The search should be functional

Reply:
@Oshioke-Salaki the search is functional now